src/main/java/org/scijava/io/handle/DataHandles.java

```diff
@@ -108,4 +111,135 @@ private static synchronized void initUTFMethod() {
 		}
 	}
+
+	/**
+	 * Copies all bytes from the input to the output handle. Reading and writing
+	 * start at the current positions of the handles.
+	 *
+	 * @param in the input handle
+	 * @param out the output handle
+	 * @return the number of bytes copied
+	 * @throws IOException
+	 */
+	public static long copy(final DataHandle<Location> in,
+		final DataHandle<Location> out) throws IOException
+	{
+		return copy(in, out, 0l, null);
+	}
+
+	/**
+	 * Copies all bytes from the input to the output handle, reporting the
+	 * progress to the provided task. Reading and writing start at the current
+	 * positions of the handles.
+	 *
+	 * @param in the input handle
+	 * @param out the output handle
+	 * @param task task to report progress to
+	 * @return the number of bytes copied
+	 * @throws IOException
+	 */
+	public static long copy(final DataHandle<Location> in,
+		final DataHandle<Location> out, final Task task) throws IOException
+	{
+		return copy(in, out, 0l, task);
+	}
+
+	/**
+	 * Copies up to <code>length</code> bytes from the input to the output handle.
+	 * Reading and writing start at the current positions of the handles. Stops
+	 * early if there are no more bytes available from the input handle.
+	 *
+	 * @param in the input handle
+	 * @param out the output handle
+	 * @param length the number of bytes to copy
+	 * @return the number of bytes copied
+	 * @throws IOException
+	 */
+	public static long copy(final DataHandle<Location> in,
+		final DataHandle<Location> out, final int length) throws IOException
+	{
+		return copy(in, out, length, null);
+	}
+
+	/**
+	 * Copies up to <code>length</code> bytes from the input to the output handle,
+	 * reporting the progress to the provided task. Reading and writing start at
+	 * the current positions of the handles. Stops early if there are no more
+	 * bytes available from the input handle.
+	 *
+	 * @param in input handle
+	 * @param out the output handle
+	 * @param length maximum number of bytes to copy, will copy all bytes if set
+	 *          to <code>0</code>
+	 * @param task a task object to use for reporting the status of the copy
+	 *          operation. Can be <code>null</code> if no reporting is needed.
+	 * @return the number of bytes copied
+	 * @throws IOException
+	 */
+	public static long copy(final DataHandle<Location> in,
+		final DataHandle<Location> out, final long length, final Task task)
+		throws IOException
+	{
+		return copy(in, out, length, task, 64 * 1024);
+	}
+
+	/**
+	 * Copies up to <code>length</code> bytes from the input to the output handle,
+	 * reporting the progress to the provided task. Reading and writing start at
+	 * the current positions of the handles. Stops early if there are no more
+	 * bytes available from the input handle. Uses a buffer of the provided size,
+	 * instead of using the default size.
+	 *
+	 * @param in input handle
+	 * @param out the output handle
+	 * @param length maximum number of bytes to copy, will copy all bytes if set
```

Comment:
More grammatically correct would be `;` instead of `,`. Otherwise it's a [comma splice](https://en.wikipedia.org/wiki/Comma_splice).